c2rust-analyze/src/rewrite/mod.rs

```diff
@@ -62,6 +62,12 @@ pub enum Rewrite<S = Span> {
     Cast(Box<Rewrite>, String),
     /// The integer literal `0`.
     LitZero,
+    /// Cell::new
+    CellNew,
```

Comment:
`CellNew` should take a `Box<Rewrite>` argument for consistency, even if it's always just `Identity` at the moment.  Also, there are some checks we plan to implement (TODO comments in `apply.rs`) that involve looking at which parts of the original expr (either `Identity` or `Sub`-exprs) appear in its `Rewrite`.

Reply:
addressed in d0d400d